src/common/util.h

```diff
@@ -46,6 +46,8 @@
 
 #include "crypto/hash.h"
 
+#define MONERO_DONATION_ADDR "44AFFq5kSiGBoZ4NMDwYtN18obc8AemS33DBLWs3H7otXft3XjrpDtQGv7SqSsaBYBb98uNbr2VBBEt7f2wfn3RVGQBEP3A"
```

Comment:
My suggestion would be to place in simplewallet.h and include said file in the unit test.

Reply:
Sounds good to me.